core/templates/dev/head/pages/base.html

```diff
@@ -84,6 +84,7 @@
           '{{can_create_collections|js_string}}'),
         csrf_token: JSON.parse('{{csrf_token|js_string}}'),
         DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}'),
+        status_code: JSON.parse('{{status_code}}' || '200'),
```

Comment:
Don't default to 200. Just pass the status code from the backend.

I think it's OK to pass 200 in the success case. Make sure to add a test for that after you've added that.

Reply:
I didn't get it. What do you mean by passing 200 in the success case and the test? Please elaborate it.
If I don't use that OR 200 the application is crashing and showing an error "Unexpected end of JSON input" when a non-error page is rendered, as the "status_code" will not be sent by the backend in that situation! @seanlip

Reply:
Yep -- the point is to try and find a way for the backend to send a 200 to the frontend in the non-error case, similar to how 400, 404, etc. are passed to the frontend in the error case.